pennylane/measurements/__init__.py

```diff
@@ -124,7 +122,7 @@ def circ(x, y):
 QNodes can be executed as usual when collecting mid-circuit measurement statistics:
 
 >>> circ(1.0, 2.0, shots=5)
```

Comment:
```suggestion
>>> qml.set_shots(circ, shots=5)(1.0, 2.0)
```
I'm surprised that wasn't warning...